frontend/packages/dev-console/src/components/topology-list/TopologyToggleIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { chart_global_Fill_Color_900 as Color900 } from '@patternfly/react-tokens';
+import { Tooltip } from '@patternfly/react-core';
+
+interface TopologyToggleIconProps {
+  url: string;
+  icon: React.ReactNode;
+  tooltipText: string;
+}
+
+const TopologyToggleIcon: React.FC<TopologyToggleIconProps> = ({ url, icon, tooltipText }) => {
+  return (
+    <Tooltip position="left" content={tooltipText}>
+      <Link to={url} style={{ color: Color900.value }}>
```

Comment:
Yes.
```suggestion
      <Button variant="link" onClick={() => history.push(url)}>
```

Reply:
This isn't a better option because the color doesn't match. Also it's best to have a link and not a button.
The current solution is ok but has no hover color.

Reply:
The link variant would create a link, and it has the hover treatments.